config/faf-clans/faf-clans.env

```diff
@@ -1,3 +1,7 @@
-WEBROOT=/usr/share/nginx/html
-TEMPLATE_NGINX_HTML=0
+API_BASE_URL=https://api.dev.faforever.com
+OAUTH_CLIENT_ID=83891c0c-feab-42e1-9ca7-515f94f808ef
+OAUTH_REDIRECT_URI=https://clans.faforever.com
```

Comment:
Locally you don't provide any env file and it work ...
But normally you use the development mode from node and not docker ^^